var/spack/repos/builtin/packages/py-bluepymm/package.py

```diff
@@ -26,6 +26,8 @@ class PyBluepymm(PythonPackage):
     depends_on('py-lxml', type='run')
     depends_on('py-sh', type='run')
     depends_on('neuron', type='run')
+    depends_on('py-h5py', type='run')
```

Comment:
we need to restrict version here `depends_on('py-h5py@:2.99', type='run')` because bluepymm still uses old API to access strings https://github.com/BlueBrain/BluePyMM/blob/bcd866f9af7f33c88fc82838ada64fd08add93cb/bluepymm/prepare_combos/parse_files.py#L213.

Reply:
Thanks, I restricted the dependency to h5py < 3.
@anilbey @wvangeit could you confirm if you are fine with these new dependencies in `py-bluepymm` (`py-h5py`, `py-pyyaml`) that I took from `setup.py` [here](https://github.com/BlueBrain/BluePyMM/blob/bcd866f9af7f33c88fc82838ada64fd08add93cb/setup.py#L30-L31)?
Should you need to restrict to h5py < 3 also in `setup.py`?

Reply:
BluePyMM works with both h5py<3 and h5py>=3. 
With this commit below if string is represented as bstr it's decoding it otherwise passing the original str. https://github.com/BlueBrain/BluePyMM/commit/b16e1652c8504bd5bf00cf6db14e9c025aebbfb8

Reply:
Sorry @GianlucaFicarelli . I missed this post-processing. It should work with both versions.